### ISSUE TITLE
Provide tag autocomplete or selection from existing tags

### ISSUE THREAD

**Issue by lacasseio** [redsolver/noteless#83]
Remembering the exact tag name can be a bit cumbersome. Providing a way to either autocomplete the tag as we type them or selecting a tag from the already existing one would be a great feature.

**Comment by wisdom420** [comment]
yes.autocompleting tag name or creating a new tag with '#" would be a nice feature.
